src/Symfony/Component/ErrorRenderer/ErrorRenderer/HtmlErrorRenderer.php

```diff
@@ -80,9 +80,11 @@ public function getBody(FlattenException $exception): string
      *
      * @internal
      */
-    public function getStylesheet(): string
+    public function getStylesheet(FlattenException $exception): string
```

Comment:
where is this argument passed? I don't see it.

Reply:
This was used for WebProfilerBundle, but I removed it because the X-Debug option doesn't make sense in that case.